service/features/service.feature

```diff
@@ -97,8 +97,8 @@ Feature: Isilon CSI interface
 
      Examples:
      | induced               | errormsg                                                           |
-     | "StatsError"          | "Could not retrieve capacity. Data returned error"         |
-     | "InstancesError"      | "runid=1 Could not retrieve capacity. Error 'Error retrieving Statistics'" |
+     | "StatsError"          | "runid=1 Could not retrieve capacity. Data returned error"         |
+     | "InstancesError"      | "runid=1 Could not retrieve capacity. Error Error retrieving Statistics" |

```

Comment:
Error word is repeating. Could you check the error formation?

Reply:
It's because of the code [https://github.com/dell/csi-powerscale/pull/274/files#diff-8120cde4a65d55a72e0ffe83dd202cadcc210138f8b1525e77ac20aa4e0c7388R1474](here) `return nil, status.Error(codes.Internal, utils.GetMessageWithRunID(runID, "Could not retrieve capacity. Error %s", err.Error()))`

The %s also has the word error in it so it looks like it's being repeated twice

Reply:
Maybe I should add a colon or add the quotations back? The quotation were removed from a lot of the other locations too.